src/main/java/com/icthh/xm/tmf/ms/activation/domain/SagaEvent.java

```diff
@@ -0,0 +1,48 @@
+package com.icthh.xm.tmf.ms.activation.domain;
+
+import com.icthh.xm.tmf.ms.activation.repository.converter.MapToStringConverter;
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+import javax.persistence.Column;
+import javax.persistence.Convert;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.PrePersist;
+import javax.persistence.PreUpdate;
+import javax.persistence.Table;
+import java.io.Serializable;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+@Data
+@Accessors(chain = true)
+@Entity
+@Table(name="saga_event")
+public class SagaEvent implements Serializable {
+
+    @Id
+    private String id = UUID.randomUUID().toString();
+    @Column(name = "type_key")
+    private String typeKey;
+    @Column(name = "tenant_key")
+    private String tenantKey;
+    @Column(name = "transaction_id")
+    private String transactionId;
+    @Convert(converter = MapToStringConverter.class)
+    @Column(name = "task_context")
+    private Map<String, Object> taskContext = new HashMap<>();
+
+    @Column(name="back_off")
```

Comment:
`name="back_off"` or  `name = "back_off"`? It is used different style. Are you using checkstyle check?